UpdatedImageProcessing/Classifiers/__init__.py

```diff
@@ -0,0 +1,5 @@
+#from classify_color import ColorClassifier
+#from classify_shape import ShapeClassifier
+#from classify_tester import ClassifierTester
+
+print("shit1")
```

Comment:
Please remove this.

Reply:
Removed in https://github.com/FlintHill/SUAS-Competition/pull/97/commits/5b99f5dba88c8915e7270ad090163a92629686c1